Pipfile

```diff
@@ -20,6 +20,11 @@ mbed-tools-ci-scripts = "*"
 pre-commit = "*"
 requests-mock = "*"
 factory_boy = "*"
+# Additional requirements for ./ci_scripts
+boto3 = "*"
+jinja2 = "*"
```

Comment:
Shouldn't this have been added to the Pipfile in the same commit that added this requirement in setup.py (i.e. in "merge mbed-build package")?

Reply:
The requirement is added in "merge mbed-targets ci_scripts", not `mbed-build`.

Reply:
OK